src/mono/wasm/Wasm.Build.Tests/Common/BuildEnvironment.cs

```diff
@@ -32,7 +32,10 @@ public class BuildEnvironment
         public static readonly string           RelativeTestAssetsPath = @"..\testassets\";
         public static readonly string           TestAssetsPath = Path.Combine(AppContext.BaseDirectory, "testassets");
         public static readonly string           TestDataPath = Path.Combine(AppContext.BaseDirectory, "data");
-        public static readonly string           TmpPath = Path.Combine(AppContext.BaseDirectory, "wbt");
+        // ActiveIssue for Linux/OSx: https://github.com/dotnet/runtime/issues/92335
+        public static readonly string           TmpPath = RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ?
+                                                    Path.Combine(AppContext.BaseDirectory, "wbt artifacts") :
```

Comment:
why do we use this path only on windows? does it currently break on the other platforms?

Reply:
Yes, I am still trying to find a solution for OSX, Linux.